src/avt/Queries/Queries/avtXRayImageQuery.h

```diff
@@ -155,7 +158,7 @@ class QUERY_API avtXRayImageQuery : public avtDatasetQuery
     double                    width, height;
     int                       nx, ny;
 
-    std::string               absVarName;  //e.g. "absorbtivity"
+    std::string               absVarName;  //e.g. "absorptivity"
```

Comment:
Should we stick the "absorbtivity" since we use that everywhere else. It seems like either could be correct, although absorptivity does seem to be a slight bit more popular in searches. Here is statement on the difference from wikidiff.

As nouns the difference between absorptivity and absorbtivity is that absorptivity is the quality of being absorptive; absorptiveness while absorbtivity is the extent to which a material absorbs light.

Reply:
Oh... I just searched it in duckduckgo and it autocorrected to absorptivity. I didn't realize both were valid! My bad; I'll change it back.